hack/env.sh

```diff
@@ -1,6 +1,7 @@
 if [ -z $SKIP_VAR_SET ]; then
         export SRIOV_CNI_IMAGE=${SRIOV_CNI_IMAGE:-ghcr.io/k8snetworkplumbingwg/sriov-cni}
         export SRIOV_INFINIBAND_CNI_IMAGE=${SRIOV_INFINIBAND_CNI_IMAGE:-ghcr.io/k8snetworkplumbingwg/ib-sriov-cni}
+        export RDMA_CNI_IMAGE=${RDMA_CNI_IMAGE:-ghcr.io/k8snetworkplumbingwg/rdma-cni}
```

Comment:
Do you want to provide a way to skip deployment of the cni? I thin this construction will always use default vaule even if env var is explicitly set to an empty string. In my PR I used a different contruction https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/4b3a309a38d419a86dc7edb0a1bbb78fb13f71ac/hack/env.sh#L5. The difference is `:-` vs `-`. With construction from my PR it is possible to explicitly set the value to an empty string.

Reply:
It makes sense. I'll update my PR

Reply:
done